### PR TITLE
Details on how to configure the log level for network costs.

### PR DESCRIPTION
**Pull request by thomasvn**

### CODE REVIEW

File: network-costs-configuration.md [hunk 1]
```diff
@@ -33,6 +33,32 @@ networkCosts:
   enabled: true
 ```
 
+## Additional configuration
+
+You can view a list of common config options [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/700cfa306c8e78bc9a1039b584769b9a0e0757d0/cost-analyzer/values.yaml#L573).
+
+### Prometheus:
+
+- If using the included Prometheus instance, the scrape is automatically configured.
```
Comment (bstuder99) [inline network-costs-configuration.md:42]: In this context, is this referring to Kubecost-bundled Prometheus?

Comment (thomasvn) [reply network-costs-configuration.md:42]: Correct! This is referring to Kubecost's bundled Prometheus

**Review by bstuder99 — APPROVED** [review]

File: network-costs-configuration.md (hunk 1 above)
Comment (thomasvn) [reply network-costs-configuration.md:42]: No worries! I agree that we should standardize on how we refer to "Kubecost-bundled Prom". I'll make the change later today. Thanks!

Comment (thomasvn) [reply network-costs-configuration.md]: @bstuder99 Fixed!

**Review by bstuder99 — APPROVED** [review]

**Pull request merged**